### ISSUE TITLE
Problem with gateLowerEdgeLevel of orifice

### ISSUE THREAD

**Issue by rhutten** [openearth/delft3dfmpy#45]
**Problem**
The gateLowerEdgeLevel of an orifice is the position of the gate door's lower edge [m AD]. This is an absolute value. However, the gateLowerEdgeLevel specified for the orifice in the D-HyDAMO code is a relative value. See code below from delft3dfmpy/converters/hydamo_to_dflowfm:
            orifices_dfm.at[idx, "schuifhoogte"] = weir_mandev.hoogteopening.to_string(
                index=False
            )

**Comment by rhutten** [comment]
@RuudHurkmans, I checked the code in more detail. See above my proposed solution.

**Comment by RuudHurkmans** [comment]
This is correct, 'hoogteopening' was treated as an absolute level, but is in the HyDAMO format relative to the crest. In the code on github, the above solution has been implemented and it will be in the next release of the installable version.

As a workaround, in the meantime, an orifice can be added in the notebook as follows:

![image](https://user-images.githubusercontent.com/59835874/204093475-74449cf0-baad-4200-8dc7-e8f0cd43b8d2.png)

Here, an absolute value (m+NAP) should be provided, such as D-HYDRO expects it.
